packages/cli/contracts/src/codegen/tables/Dynamics1.sol

```diff
@@ -32,9 +36,7 @@ struct Dynamics1Data {
 library Dynamics1 {
   /** Get the table values' field layout */
   function getFieldLayout() internal pure returns (FieldLayout) {
```

Comment:
Do we need this method anymore?

Reply:
seems fine for completeness (so the methods match what you can get from the `Tables` table), if it's not used anywhere it's also not included in the bytecode

Reply:
I don't know why anyone would choose the less efficient one (function call) over the more efficient one though (importing the `_fieldLayout` directly)